src/main/java/org/example/work/work.java

```diff
@@ -28,18 +31,204 @@ public static void main(String[] args) {
 
         i = answer1_b1;
         s = 0;
-        while(i<=answer1_b2) {
+        while (i <= answer1_b2) {
             s = s + i;
             i++;
         }
+        System.out.println(s);
 
         //2-c
-        System.out.println("c. 숫자를 입력받고, 해당 숫자의 크기의 마름모를 출력하는 프로그램을 만들어주세요.");
-        System.out.println("숫자를 입력하세요.");
+        System.out.println("c. 숫자를 입력받고, 해당 숫자 크기의 마름모를 출력하는 프로그램을 만들어주세요.");
+        System.out.println("숫자를 입력하세요. (1 이상의 홀수)");
         int answer2_c = scan.nextInt();
+        for (int f = 0; f < 1; ) {
+            if (answer2_c < 1) {
+                System.out.println("1 이상의 홀수를 입력해주세요.");
+                answer2_c = scan.nextInt();
+                continue;
+            } else if (answer2_c > 1) {
+                if (answer2_c % 2 == 1) {
+                    break;
+                } else {
+                    System.out.println("1 이상의 홀수를 입력해주세요.");
+                    answer2_c = scan.nextInt();
+                    continue;
+                }
+            }
+        }
 
-        i = 0;
-        s = 0;
+        int n = answer2_c;
+        int m = 0;
+        int j = 0;
+        int a = 0;
+        int b = 0;
+
+        for (m = 0; n - 2 * m > 0; m++) {
+            for (b = (n + 1) / 2 - m; b > 0; b--) {
+                System.out.print(" ");
+            }
+            for (a = 2 * m - 1; a > 0; a--) {
+                System.out.print("*");
+            }
+            System.out.println("");
+        }
```

Comment:
```suggestion
        for (m = 0; n - 2 * m > 0; m++) {
            int leftNumberOfWhiteSpace = (n + 1) / 2 - m;
            for (b = leftNumberOfWhiteSpace; b > 0; b--) {
                System.out.print(" ");
            }
            int leftNumberOfStar = 2 * m - 1;
            for (a = leftNumberOfStar; a > 0; a--) {
                System.out.print("*");
            }
            System.out.println("");
        }
```
위 for문에서 사용된 수식들을 변수로 의미를 쉽게 알아챌 수 있도록 만들어봅시다.

변수명이 조금 길어도 괜찮습니다.
한번에 알아 볼 수 있는 가독성 높은 코드가 더 좋아요.